pro/src/components/pages/Offerers/Offerer/VenueV1/fields/PricingPoint/PricingPoint.tsx

```diff
@@ -49,6 +54,35 @@ const PricingPoint = ({
         <h2 className="main-list-title-text">Barème de remboursement</h2>
       </div>
 
+      {isConfirmSiretDialogOpen && (
+        <ConfirmDialog
```

Comment:
J'aurais bien fait un composant ici, ca évite d'avoir 30 lignes pour une pop-in, même si ce composant ne sera sans doute pas utilisé ailleurs.